src/libsync/propagateuploadv1.cpp

```diff
@@ -363,8 +363,10 @@ void PropagateUploadFileV1::abort(PropagatorJob::AbortType abortType)
             // dont abort final PUT which uploaded its data,
             // since this might result in conflicts
             if (PUTFileJob *putJob = qobject_cast<PUTFileJob *>(job)){
-                if (abortType == AbortType::Asynchronous && (((_currentChunk + _startChunk) % _chunkCount) == 0)
-                        && putJob->device()->atEnd()) {
+                if (abortType == AbortType::Asynchronous
+                    && _chunkCount > 0
```

Comment:
I wonder how chunk count can be 0

Reply:
`chunkCount` gets set by `doStartUpload` and the job's `abort` can be called before we get there.